### PR TITLE
Profile feed

### PR DESCRIPTION
**Pull request by Palakkgoyal**
This PR closes issue no. #481
Created a new profile feed section for profile page

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -183,8 +185,8 @@ function App() {
       variant: "info",
     });
   };
-
   return (
+    <UserContext.Provider value={user}>
```
Comment (narayan954) [inline src/App.jsx:189]: hii, you don't need to use user context, you can directly access/get the current user from firebase. Check other components , how they're getting current user from auth object

**Review by narayan954 — CHANGES_REQUESTED** [review]
some changes

File: src/App.jsx (hunk 1 above)
Comment (narayan954) [reply src/App.jsx:189]: this will resolve the merge conflict as well, automatically

Comment (Palakkgoyal) [reply src/App.jsx:189]: Okay.

**Pull request closed without merging**